main_app/templates/main_app/common.html

```diff
@@ -354,40 +405,39 @@ <h4 class="new">&nbsp;&nbsp; More</h4>
-        padding-bottom: 3%;
-        padding-top: 3%;
-        margin-top: auto;
-      "
-    >
-      <div class="row" style="margin: 0px">
-        <div class="col l6 s12" style="margin-left: 15%">
-          <h2 class="white-text"><b>Rescue</b></h2>
-          <p class="white-text text-lighten-4" style="color: white"></p>
-          <h3 style="color: white">
-            We are here to help women against crimes . and to aware about laws
-            for women
-          </h3>
-        </div>
-        <div class="row" style="margin-right: 10%">
-          <ul class="col" style="margin-top: 5px">
-            <li style="margin: 10px">
-              <a class="white-text" href="/developers">
-                <h3>Developers</h3>
-              </a>
-            </li>
-            <li style="margin: 10px">
-              <a class="white-text" href="/helpline_numbers">
-                <h3>Helpline No.</h3>
-              </a>
-            </li>
-          </ul>
+    <footer class="footer" id="footer">
+      <div class="row text-center text-xs-center text-sm-left text-md-left">
+       <div class="col-xs-12 col-sm-5 col-md-5" style="padding-right:60px;padding-left:40px;">
+         <h2 class="white-text"><b>Rescue</b></h2>
+         <p style="color: white;font-size: 20px;">We are here to help women against crimes  and to aware about laws for women</p>
+         <ul style="padding-top:10px;" class="quick-links">
+           <li><a class="white-text" href="/developers"><i class="bi bi-people-fill" style="padding-right:8px;"></i>Developers</a></li>
+         </ul>
+       </div>
+       <div  class="col-xs-12 col-sm-3 col-md-3"   style="text-decoration: none;"  class="list-unstyled quick-links">
+           <h3 style="border-left: 5px solid white;padding:8px;">Quick Links</h3>
+           <br/>
+           <ul class="list-unstyled quick-links">
+             <li><a href="/home"><i class="bi bi-house-door-fill"></i>Home</a></li>
+             <li><a href="/emergency"><i class="bi bi-exclamation-triangle-fill"></i>Emergency</a></li>
+             <li><a href="/helpline_numbers"><i class="bi bi-telephone-fill"></i>Helpline Numbers</a></li>
+             <li><a href="/emergency_contact"><i class="bi bi-person-lines-fill"></i>Emergency Contacts</a></li>
+           </ul>
+       </div>
+        <div class="col-xs-12 col-sm-4 col-md-4" style="padding-left:20px;">
+          <h3 style="border-left: 5px solid white;padding:8px;margin-left:10px;">Contact us</h3>
+          <br/><br/>
+          <a href="#"><i style="background: #55ACEE;" class="bi bi-twitter ic"></i></a>
+          <a href="#"><i style="background: #3B5998;" class="bi bi-facebook ic"></i></a>
+          <a href="#"><i style="background: #007bb5;" class="bi bi-linkedin ic"></i></a>
+          <a href="#"><i style="background: salmon;" class="bi bi-envelope-fill ic"></i></a>
+          <a href="#"><i style="background:purple;" class="bi bi bi-github ic"></i></a>
```

Comment:
Change the background color of the GitHub icon to black and for now, add the GitHub link of the `rescue` project to all the icons.

Reply:
Okay @sakshikhachane  i will make all the changes